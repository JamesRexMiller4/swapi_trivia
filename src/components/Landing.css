@font-face {
  font-family: 'StarWars Main';
  src: url('../Fonts/Starjedi.ttf');
}

#landing-main {
  background-image: url('../images/Galaxy1.jpg');
  background-position: center;
  background-repeat: no-repeat;
  background-size: cover;
  height: 100vh;
  overflow: auto;
  padding-left: 25%;
  padding-right: 25%;
  padding-top: 10vh;
  width: 100%;
}

h1 {
  color: #EEDB00;
  font-family: 'StarWars Main';
  font-size: 8rem;
  width: 100%;
}
