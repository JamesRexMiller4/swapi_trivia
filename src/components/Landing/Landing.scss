@import '../variables.scss';

#landing-main {
  background-image: url('../../images/Galaxy1.jpg');
  background-position: center;
  background-repeat: no-repeat;
  background-size: cover;
  height: 100vh;
  overflow: auto;
  padding-left: 30%;
  padding-right: 30%;
  padding-top: 3vh;
  width: 100%;
}

h1 {
  color: #EEDB00;
  font-family: $large-text-font;
  font-size: 7rem;
  line-height: 100%;
  margin-bottom: 7%;
  width: 100%;
}
