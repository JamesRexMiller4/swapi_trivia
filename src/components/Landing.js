import React from 'react';
import './Landing.css';
import Form from './Form.js';

const Landing = () => {
  return (
    <div id="landing-main">
      <h1>Swapi</h1>
      <Form />
    </div>
  );
}


export default Landing;
