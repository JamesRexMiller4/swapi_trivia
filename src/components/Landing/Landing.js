import React from 'react';
import './Landing.scss';
import Form from '../Form/Form.js';

const Landing = (props) => {
  return (
    <div id="landing-main">
      <h1>Swapi<br/>WarS</h1>
      <Form updateLogin={props.updateLogin}/>
    </div>
  );
}


export default Landing;
