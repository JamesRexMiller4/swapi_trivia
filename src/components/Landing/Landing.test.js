import React from 'react';
import ReactDOM from 'react-dom';
import Landing from './Landing';
import {shallow} from 'enzyme';

describe('Landing', () => {
  let wrapper, mockLoginUpdate;

  beforeEach(() => {
    mockLoginUpdate = jest.fn()

    wrapper = shallow(<Landing 
      mockLoginUpdate={mockLoginUpdate}
    />)
  })

  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Landing />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('should match the snapshot with all the data passed through', () => {
    expect(wrapper).toMatchSnapshot()
  });

});