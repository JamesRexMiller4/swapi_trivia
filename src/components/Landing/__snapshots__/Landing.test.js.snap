// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing should match the snapshot with all the data passed through 1`] = `
<div
  id="landing-main"
>
  <h1>
    Swapi
    <br />
    WarS
  </h1>
  <Form />
</div>
`;
